internal/pipe/webhook/webhook.go

```diff
@@ -107,13 +117,13 @@ func (p Pipe) Announce(ctx *context.Context) error {
 	}
 	defer resp.Body.Close()
 
-	switch resp.StatusCode {
-	case http.StatusOK, http.StatusCreated, http.StatusAccepted, http.StatusNoContent:
-		log.Infof("Post OK: '%v'", resp.StatusCode)
-		body, _ := io.ReadAll(resp.Body)
-		log.Infof("Response : %v\n", string(body))
-		return nil
-	default:
+	if !slices.Contains(expectedStatusCodes, resp.StatusCode) {
+		io.Copy(io.Discard, resp.Body)
```

Comment:
It's proper that in any case, we should fully read the body. So I added this in the failure case so we at least do read it, but just discard it.

I had considered, and would be willing to change, we log the response body always.

The initial concern I had with that is we'd want to likely make sure we truncate a response to some reasonable length to avoid some error page with a multi MB response page or something.

Reply:
i believe this is good enough for now, we weren't logging it before anyway!